source/getting-started/index.markdown

```diff
@@ -284,11 +284,13 @@ For additional help, in addition to this site, there are four sources:
  - [Gitter Chatroom](https://gitter.im/balloob/home-assistant) for general Home Assistant discussions and questions.
  - [GitHub Page](https://github.com/balloob/home-assistant/issues) for issue reporting.
 
+### Updating
+To update Home Assistant to the latest release run: `pip3 install --upgrade homeassistant`  
+You then have to restart Home Assistant for the changes to take effect. If you have installed it as an autostarting daemon (see below), then run: `sudo service hass-daemon restart` 
```

Comment:
I think that we should not add a daemon-specific command here. From my point if a user is using some kind of autostarting mechanism she/he is aware of the commands to start/stop/restart the used daemon or init implementation.